examples/enumkeys.rs

```diff
@@ -0,0 +1,50 @@
+use bevy::{log::LogPlugin, prelude::*};
+use bevy_pkv::PkvStore;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+struct User {
+    name: String,
+}
+
+fn setup(mut pkv: ResMut<PkvStore>) {
+    // strings
+    if let Ok(username) = pkv.get::<String>("username") {
+        info!("Welcome back {username}");
+    } else {
+        pkv.set_string("username", "alice")
```

Comment:
Isn't this unrelated to the new feature?

Reply:
it is related I just did not see it to change it to the enum. I just copyed the other example and missed the set_string method